src/store/lookup.ts

```diff
@@ -477,6 +481,75 @@ async function lookupCardInStock(store: Store, page: Page, link: Link) {
 	return true;
 }
 
+async function runCaptchaDeterrent(browser: Browser, store: Store, page: Page) {
+	const successStatusCodes = store.successStatusCodes ?? [[0, 399]];
+	let statusCode = 0;
+	let deterrentLinks: string[] = [];
+
+	logger.debug(`[${store.name}] Navigating to random anti-captcha page...`);
+
+	if (store.captchaDeterrent?.hardLinks?.length) {
+		deterrentLinks = deterrentLinks.concat(
+			store.captchaDeterrent.hardLinks
+		);
+	}
+
+	if (store.captchaDeterrent?.searchUrl) {
+		if (store.captchaDeterrent.searchTerms) {
+			store.captchaDeterrent.searchTerms.forEach((element) =>
+				deterrentLinks.push(
+					store.captchaDeterrent?.searchUrl
+						? store.captchaDeterrent.searchUrl.replace(
+								'%%s',
+								element
+						  )
+						: ''
+				)
+			);
+		}
+	}
+
+	if (deterrentLinks.length > 0) {
+		const link: Link = {
+			brand: 'captcha-deterrent',
+			model: 'captcha-deterrent',
+			series: 'captcha-deterrent',
+			url:
+				deterrentLinks[
+					Math.floor(Math.random() * deterrentLinks.length)
+				]
+		};
+		logger.debug(`Selected captcha-deterrent link: ${link.url}`);
+
+		try {
+			const givenWaitFor = store.waitUntil
+				? store.waitUntil
+				: 'networkidle0';
+			const response: Response | null = await page.goto(link.url, {
+				waitUntil: givenWaitFor
+			});
+			statusCode = await handleResponse(
+				browser,
+				store,
+				page,
+				link,
+				response
+			);
+			setTimeout(() => {
+				// Do nothing
+			}, 3000);
```

Comment:
How come we want this 3 second timeout?

Reply:
This is basically an arbitrary "wait after page load so we don't look as bot-like". I was being cautious about us hitting a search page and then immediately bouncing to another page as soon as the search results are loaded. I wanted to at least KINDA look like someone actually viewed the search results hehe.

Reply:
That makes sense to me :)